services/web/client/source/class/osparc/component/service/manager/ActivityManager.js

```diff
@@ -99,52 +99,40 @@ qx.Class.define("osparc.component.service.manager.ActivityManager", {
      */
     __createActionsBar: function() {
       const toolbar = new qx.ui.toolbar.ToolBar();
+      const tablePart = new qx.ui.toolbar.Part();
       const actionsPart = new qx.ui.toolbar.Part();
-      toolbar.addSpacer();
-      toolbar.add(actionsPart);
+      toolbar.add(tablePart);
+      // toolbar.addSpacer();
+      // toolbar.add(actionsPart);
+
+      const reloadButton = this.__reloadButton = new qx.ui.toolbar.Button(this.tr("Reload"), "@FontAwesome5Solid/sync-alt/14");
+      tablePart.add(reloadButton);
+      reloadButton.addListener("execute", () => {
+        this.__tree.exclude();
+        this.__fetchingView.show();
+        this.__tree.reset().then(() => {
+          this.__tree.show();
+          this.__fetchingView.exclude();
+        });
+      }, this);
 
       const runButton = new qx.ui.toolbar.Button(this.tr("Run"), "@FontAwesome5Solid/play/14");
       actionsPart.add(runButton);
+      runButton.addListener("execute", () => osparc.component.message.FlashMessenger.getInstance().logAs("Not implemented"));
 
       const stopButton = new qx.ui.toolbar.Button(this.tr("Stop"), "@FontAwesome5Solid/stop-circle/14");
       actionsPart.add(stopButton);
+      stopButton.addListener("execute", () => osparc.component.message.FlashMessenger.getInstance().logAs("Not implemented"));
 
       const infoButton = new qx.ui.toolbar.Button(this.tr("Info"), "@FontAwesome5Solid/info/14");
       actionsPart.add(infoButton);
+      infoButton.addListener("execute", () => osparc.component.message.FlashMessenger.getInstance().logAs("Not implemented"));
```

Comment:
would it not make more sense to remove/hide these buttons if they are not implemented?

Reply:
This buttons are already hidden, you won't see them if you test the interface. I will comment out the code!